src/App.js

```diff
@@ -20,4 +32,5 @@ const BodyStyled = styled.div`
   display: grid;
```

Comment:
Willst du diesen layout auf andere Seiten auch benutzen? Wenn ja, vielleicht könntest du das in deinem GlobalStyles speichern.

Reply:
Ich bin mir noch nicht sicher, ob ich das für alle Seiten so übernehmen möchte, deshalb lasse ich es erst noch hier ind er App..